.github/workflows/update_version.yml

```diff
@@ -0,0 +1,54 @@
+name: Update version in all workspaces
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_name:
+        type: string
+        description: XX_X (23_2+)
+        required: true
+        default: 23_2
+      version:
+        type: string
+        description: XX.X.X Set exact version (optional)
+        required: true
+        default: 23.2.1
+
+jobs:
+  update_version:
+    name: Update version and create PR
+    runs-on: ubuntu-latest
```

Comment:
Don't we need to use self-hosted runner?

Reply:
In this action uses the GitHub CLI [command](https://github.com/DevExpress/DevExtreme/pull/25842/files/52ff15a9b1c7f6d8ec14928f5c12ba5ad6244f12#diff-8d4c99794a132146cc00f3b4860b4896f1627711754c64088d1005786ab3f0d9R53), but it is not installed in the self-hosted runner, so we use github-hosted runner.